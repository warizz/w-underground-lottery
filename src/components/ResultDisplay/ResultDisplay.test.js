import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Uut from './';

describe('snapshot testing', () => {
  it('should match snapshot when no props', () => {
    const wrapper = shallow(<Uut />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });

  it('should match snapshot with result', () => {
    const props = {
      firstThree: '123',
      fourthThree: '101',
      secondThree: '456',
      six: '123456',
      thirdThree: '789',
      two: '69',
    };
    const wrapper = shallow(<Uut {...props} />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
