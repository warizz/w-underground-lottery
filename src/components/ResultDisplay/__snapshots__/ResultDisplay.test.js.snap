// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing should match snapshot when no props 1`] = `
<SegmentGroup
  className="ResultDisplay"
  size="massive"
>
  <Segment
    color="green"
    textAlign="center"
  >
    -
  </Segment>
  <Segment
    textAlign="center"
  >
    -
  </Segment>
  <SegmentGroup
    horizontal={true}
  >
    <Segment
      textAlign="center"
    >
      -
    </Segment>
    <Segment
      textAlign="center"
    >
      -
    </Segment>
  </SegmentGroup>
  <SegmentGroup
    horizontal={true}
  >
    <Segment
      textAlign="center"
    >
      -
    </Segment>
    <Segment
      textAlign="center"
    >
      -
    </Segment>
  </SegmentGroup>
</SegmentGroup>
`;

exports[`snapshot testing should match snapshot with result 1`] = `
<SegmentGroup
  className="ResultDisplay"
  size="massive"
>
  <Segment
    color="green"
    textAlign="center"
  >
    123456
  </Segment>
  <Segment
    textAlign="center"
  >
    69
  </Segment>
  <SegmentGroup
    horizontal={true}
  >
    <Segment
      textAlign="center"
    >
      123
    </Segment>
    <Segment
      textAlign="center"
    >
      456
    </Segment>
  </SegmentGroup>
  <SegmentGroup
    horizontal={true}
  >
    <Segment
      textAlign="center"
    >
      789
    </Segment>
    <Segment
      textAlign="center"
    >
      101
    </Segment>
  </SegmentGroup>
</SegmentGroup>
`;
